main.go

```diff
@@ -282,6 +304,34 @@ func setupFilter(filters []string) []glob.Glob {
 	return matchers
 }
 
+func setupDiscordFormat(discordFormat map[string]string) (map[string]string, error) {
+	var err error
+	// lowercase to match that YAML lowercases it
+	discordFormatDefaults := map[string]string{
+		"pm":   "${SERVER},${NICK}!${IDENT}@${HOST} - ${NICK}@${DISCRIMINATOR}: ${CONTENT}",
+		"join": "_${NICK} joined (${IDENT}@${HOST})_",
+		"part": "_${NICK} left (${IDENT}@${HOST}) - ${CONTENT}_",
+		"quit": "_${NICK} quit (${IDENT}@${HOST}) - Quit: ${CONTENT}_",
+		"kick": "_${TARGET} was kicked by ${NICK} - ${CONTENT}_",
+		"nick": "_${NICK} changed nick to ${CONTENT}_",
+	}
```

Comment:
I'd still like the default to be _off_ (empty string), so maybe we should move this to config.yml, but commented out.

Additionally, please could you try to match the original format that was created programmatically, if possible? The original ones were designed to match irccloud's output 1:1, which I believe to be a reasonable default for good UX.